core/tx_pool_test.go

```diff
@@ -261,6 +261,52 @@ func TestInvalidTransactions(t *testing.T) {
 	}
 }
 
+func TestBlacklistedTransactions(t *testing.T) {
+	// DO NOT parallelize, test will add accounts to tx pool config used by other tests.
+
+	// Create the pool
+	pool, _ := setupTxPool()
+	defer pool.Stop()
+
+	// Create testing keys
+	bannedFromKey, _ := crypto.GenerateKey()
+	goodFromKey, _ := crypto.GenerateKey()
+
+	// Create testing transactions
+	badTx := transaction(0, 25000, bannedFromKey)
+	goodTx := transaction(0, 25000, goodFromKey)
+	bannedFromAcc, _ := deriveSender(badTx)
+	bannedToAcc := *badTx.To()
+	goodFromAcc, _ := deriveSender(goodTx)
+
+	// Fund from accounts
+	pool.currentState.AddBalance(bannedFromAcc, big.NewInt(50100))
+	pool.currentState.AddBalance(goodFromAcc, big.NewInt(50100))
+
+	(*DefaultTxPoolConfig.Blacklist)[bannedFromAcc] = false
+	(*DefaultTxPoolConfig.Blacklist)[bannedToAcc] = true
```

Comment:
it should be commented somewhere about the format of the blacklist file (that false => from blacklist, true => to blacklist)

Reply:
yes, will add to usage msg